setup.py

```diff
@@ -342,6 +342,13 @@ def report(*args):
 cmake = CMake()
 
 
+def is_source_hipified():
```

Comment:
Nit
```suggestion
def is_source_hipified() -> bool:
```

Reply:
Fixed in https://github.com/pytorch/pytorch/pull/114309/commits/9f028458523aefc1dc09612eff095e2ea416921f